en/node-operators-guide/request-response.md

```diff
@@ -90,14 +90,55 @@ Errors and warnings emitted by the **Orakl Network Request-Response** can be [se
 
 Before launching the Request-Response solution, the **Orakl Network API** has to be accessible from the **Orakl Network Request-Response** to load listener settings.
 
-After the **Orakl Network API** is healthy, Request-Response microservices (listener, worker, reporter) can be launched in an arbitrary order. Microservices communicate with each other through the BullMQ - job queue.
+After the **Orakl Network API** is healthy, launch the VRF service, which consists of listener, worker, and reporter microservices, with the command below. Microservices communicate with each other through the BullMQ - job queue.
+
+```sh
+yarn start:core:request_response
+```
+
+For development purposes, the service can be run with `nodemon`, which automatically restarts all the microservices when file changes are detected, using the following command:
```

Comment:
how about making it shorter?
```
run in dev mode through following command
```

Reply:
good point. I thought it might be useful to mention we are using nodemon but i guess if anyone's interested they can figure out by looking in to the package.json